data/types/blood/glucose/summary/summary.go

```diff
@@ -0,0 +1,28 @@
+package summary
+
+import (
+    "time"
+    "go.mongodb.org/mongo-driver/bson/primitive"
+	"github.com/tidepool-org/platform/data/types/blood/glucose"
+)
+
+type Summary struct {
+    ID                 primitive.ObjectID  `json:"-" bson:"_id,omitempty"`
+    UserID             string              `json:"userId" bson:"_userId"`
+
+    // date tracking
+    LastUpdated        time.Time   `json:"lastUpdated" bson:"lastUpdated"`
+    FirstData          time.Time   `json:"firstData" bson:"firstData"`
+    LastData           time.Time   `json:"lastData" bson:"lastData"`
+    LastUpload         time.Time   `json:"lastUpload" bson:"lastUpload"`
+
+    // actual values
+    AverageGlucose     glucose.Glucose     `json:"avgGlucose" bson:"avgGlucose"`
+    TimeInRange        float64             `json:"timeInRange" bson:"timeInRange"`
```

Comment:
Should this be a pointer? When a summary is not yet calculated this will be set to `0` which is different from all datums being out of range.